R/figure.R

```diff
@@ -11,6 +11,7 @@ fig_save <- function(plot,
                      fig.width = 7.29, # 700 pixels
                      fig.height = NULL,
                      fig.retina = 2,
+                     fig.pointsize = 72,
```

Comment:
This should be 12 by default, and then the graphics device should get `pointsize = pointsize * fig.retina`